sympy/physics/hep/tests/test_gamma_matrices.py

```diff
@@ -399,3 +400,25 @@ def test_gamma_matrix_trace():
     t = ps*ps*ps*ps*ps*ps*ps*ps
     r = gamma_trace(t)
     assert r.equals(4*p2*p2*p2*p2)
+
+def test_bug_13636():
```

Comment:
Usually there should be two blank lines between top-level functions, classes etc.

Reply:
Thanks.

Reply:
This is fixed now.